dev/tests/unit/Magento/FunctionalTestFramework/Util/GenerationErrorHandlerTest.php

```diff
@@ -21,10 +22,9 @@ class GenerationErrorHandlerTest extends MagentoTestCase
      */
     public function testGetDistinctErrors()
     {
-        AspectMock::double(
-            MftfApplicationConfig::class,
-            ['getPhase' => MftfApplicationConfig::GENERATION_PHASE]
-        );
+        $this->createMock(MftfApplicationConfig::class)
+            ->method('getPhase')
+            ->willReturn(MftfApplicationConfig::GENERATION_PHASE);
```

Comment:
@anzin can you please explain the approach? How does this mocking work if the create mock object is not used anywhere? Can this code be simply removed?

cc @jilu1 @soumyau

Reply:
@sivaschenko Thank you, for your suggestion. I had doubts about that. I will just remove that.